apps/filebrowser/src/filebrowser/urls.py

```diff
@@ -44,6 +44,8 @@
   # POST operations
   re_path(r'^save$', filebrowser_views.save_file, name="filebrowser_views_save_file"),
   re_path(r'^upload/file$', filebrowser_views.upload_file, name='upload_file'),
+  re_path(r'^upload/chunks', filebrowser_views.upload_chunks, name='upload_chunks'),
```

Comment:
How is the sequential order of these 2 kinds of request calls? When upload complete, how it ensure all chunks are uploaded?